### DISCUSSION TITLE
add visibility to OpenweatherMap weather more-info

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#7674] [Dashboard] [answered]
please add attribute `visibility` to the `weather.openweathermap` more-info to have that available at first glance. And make it  align with other weather integrations, like Buienradar, Darksky, and even custom eg Weatherbit:

<img width="396" alt="Schermafbeelding 2020-11-13 om 09 09 13" src="https://user-images.githubusercontent.com/33354141/99044653-63834e00-2590-11eb-92c1-d95cd0459758.png">

vs

<img width="403" alt="Schermafbeelding 2020-11-13 om 09 09 00" src="https://user-images.githubusercontent.com/33354141/99044689-74cc5a80-2590-11eb-803c-5d53377386e3.png">

<img width="389" alt="Schermafbeelding 2020-11-13 om 09 09 21" src="https://user-images.githubusercontent.com/33354141/99044691-75fd8780-2590-11eb-9392-876b7d1515e8.png">

<img width="397" alt="Schermafbeelding 2020-11-13 om 09 09 29" src="https://user-images.githubusercontent.com/33354141/99044694-75fd8780-2590-11eb-83e0-af2cbb654d21.png">

**Comment by zsarnett** [comment, marked as answer]
Does it have this attribute?

**Reply by zsarnett** [reply]
It doesn't. This is a backend Feature Request

**Comment by Mariusthvdb** [comment]
well I'll be ...
you're right, it doesn't.. making this a backend FR  I fear?

sorry I didn't check that, because I have the visibility in the list for the integration, I didn't even consider this to be missing in the weather entity.

guess will need to cross the Chinese Wall of HA development.... ;-)
